src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -373,6 +373,30 @@ public sealed override bool HasUnsupportedMetadata
 
         #endregion
 
+        /// <summary>
+        /// Is this a field of a tuple type?
+        /// </summary>
```

Comment:
I think it is worth commenting on distinction between proper tuple element and some random field that happened to be on the ValueTuple<>
From the usage it seems that this is true only for elements, but I am not sure if I am inferring that correctly.